CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
-DEVELOPING
+CHANGES IN VERSION 1.12.1 (FROM 1.12.0)
 ===================================
 
-#### General
-* Added `defaultImageTemplateScale` to arrange image comparison logic [#307](https://github.com/appium/appium-base-driver/pull/307). Read [doc](docs/en/advanced-concepts/image-elements.md) for more details.
```

Comment:
can we bump https://github.com/appium/appium-base-driver/commits/master ?
(It includes only one fix for _find by image_ which has been fixed partially in 1.12.0)

Reply:
I forgot about that. Do you mind if we do a patch release for it at a later date (near future)? We're testing the rc right now so we can support iOS 12.2 and I'd rather not introduce another dependency.

Reply:
Okay. I don't mind that way. Thanks